src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -469,13 +469,10 @@ object DepositHandler {
   }
 
   def isOnPosixFileSystem(file: File): Boolean = {
-    try {
+    Try {
```

Comment:
good catch! That's better indeed.
**challenge**: you could even make this into a one-liner if you want. hint: use [Try.fold](https://www.scala-lang.org/api/current/scala/util/Try.html#fold[U](fa:Throwable=%3EU,fb:T=%3EU):U)

Reply:
Try(Files.getPosixFilePermissions(file.toPath)).fold(_ => false, _ => true)?

Reply:
you rock! 👍

Reply:
Thanks for the tip!